### DISCUSSION TITLE
How to run hermes without react-native

### DISCUSSION THREAD

**Discussion by ayroblu** [facebook/hermes#1579] [Q&A]
Hey, I was interested in trying to use hermes on its own, followed the instructions on https://github.com/facebook/hermes/blob/main/doc/CrossCompilation.md to get a build output.
But I'm not sure how to use the resulting aar package in my code?
The only api I see in classes.jar is com.facebook.hermes.BuildConfig.

I'm not really sure how to proceed, would love any pointers

**Comment by tmikov** [comment]
Hi, sorry there aren't Android-specific tutorials about this. However https://github.com/tmikov/hermes-jsi-demos is an example of using Hermes without RN.
